<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_31">
      <item row="0" column="0">
       <widget class="QDockWidget" name="scanning_widget">
        <property name="windowTitle">
         <string>Scanning control</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents_9">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>APD 1</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_16">
             <item row="0" column="0">
              <widget class="PlotWidgetModified" name="apd1_xy_ViewWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>256</width>
                 <height>256</height>
                </size>
               </property>
               <property name="transformationAnchor">
                <enum>QGraphicsView::AnchorUnderMouse</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,1,0,0,0">
               <item>
                <widget class="ScienDSpinBox" name="apd1_xy_cb_max_DoubleSpinBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="acceptDrops">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="accelerated">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string>c/s</string>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="minimum">
                  <double>-1000000000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>1000000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>100000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ScienDSpinBox" name="apd1_xy_cb_high_percentile_DoubleSpinBox">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="suffix">
                  <string>%</string>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>100.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="PlotWidget" name="apd1_xy_cb_ViewWidget">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ScienDSpinBox" name="apd1_xy_cb_low_percentile_DoubleSpinBox">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="suffix">
                  <string>%</string>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <double>99.989999999999995</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ScienDSpinBox" name="apd1_xy_cb_min_DoubleSpinBox">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="acceptDrops">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="accelerated">
                  <bool>true</bool>
                 </property>
                 <property name="suffix">
                  <string>c/s</string>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="minimum">
                  <double>-1000000000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>1000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="apd1_image_reset_pushButton">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>APD 2</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_17">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="1,0">
               <item>
                <widget class="PlotWidgetModified" name="apd2_xy_ViewWidget">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>256</width>
                   <height>256</height>
                  </size>
                 </property>
                 <property name="transformationAnchor">
                  <enum>QGraphicsView::AnchorUnderMouse</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,0,1,0,0,0">
                 <item>
                  <widget class="ScienDSpinBox" name="apd2_xy_cb_max_DoubleSpinBox">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="acceptDrops">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="suffix">
                    <string>c/s</string>
                   </property>
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>-1000000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>1000000000.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>100000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="ScienDSpinBox" name="apd2_xy_cb_high_percentile_DoubleSpinBox">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="suffix">
                    <string>%</string>
                   </property>
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>0.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>100.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>100.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="PlotWidget" name="apd2_xy_cb_ViewWidget">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>80</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="ScienDSpinBox" name="apd2_xy_cb_low_percentile_DoubleSpinBox">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="suffix">
                    <string>%</string>
                   </property>
                   <property name="maximum">
                    <double>99.989999999999995</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="ScienDSpinBox" name="apd2_xy_cb_min_DoubleSpinBox">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="mouseTracking">
                    <bool>true</bool>
                   </property>
                   <property name="acceptDrops">
                    <bool>true</bool>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                   <property name="accelerated">
                    <bool>true</bool>
                   </property>
                   <property name="suffix">
                    <string>c/s</string>
                   </property>
                   <property name="decimals">
                    <number>2</number>
                   </property>
                   <property name="minimum">
                    <double>-1000000000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>10000000.000000000000000</double>
                   </property>
                   <property name="value">
                    <double>1000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="apd2_image_reset_pushButton">
                   <property name="text">
                    <string>Reset</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Scan range [µm]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="xy_scan_range_spinBox">
                <property name="maximum">
                 <number>1000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string>Integration time [s]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="apd_integration_time_doubleSpinBox"/>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="xy_scan_pushButton">
              <property name="text">
               <string>XY scan</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>Scan step [µm]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="xy_scan_step_doubleSpinBox">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="save_scan_data_pushButton">
              <property name="text">
               <string>Save Data</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ScienDSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qtwidgets.scientific_spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>PlotWidgetModified</class>
   <extends>QGraphicsView</extends>
   <header>qtwidgets.plotwidget_modified.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
